sdk/src/lib.rs

```diff
@@ -346,6 +346,7 @@ pub trait Namada: Sized + MaybeSync + MaybeSend {
         description: Option<String>,
         website: Option<String>,
         discord_handle: Option<String>,
+        avatar: Option<String>,
```

Comment:
Not strictly necessary, but we could remove the optional `email`, `description`, `website`, `discord_handle`, `avatar`, and `commission_rate` arguments from here so that `new_change_metadata` is less cumbersome to call. Then we could augment the `args::MetaDataChange` builder to allow setting these properties individually. See the builder for `TxIbcTransfer` which has lots of optional parameters for instance.